srcpkgs/ucspi-ssl/template

```diff
@@ -1,25 +1,33 @@
 # Template file for 'ucspi-ssl'
 pkgname=ucspi-ssl
-version=0.99d
-revision=6
-short_desc="Command-line tools for building SSL client-server applications"
-maintainer="Evan Deaubl <evan@deaubl.name>"
-license="Public Domain"
+version=0.99e
+revision=1
+wrksrc="${pkgname}-${version}"
+create_wrksrc=yes
+build_wrksrc="host/superscript.com/net/${pkgname}-${version}/src"
 build_style=gnu-makefile
-hostmakedepends="perl libressl-devel"
+hostmakedepends="wget perl libressl-devel"
 makedepends="libressl-devel"
 depends="ucspi-tcp"
+short_desc="Command-line tools for building SSL client-server applications"
+maintainer="Evan Deaubl <evan@deaubl.name>"
+license="Public Domain"
 homepage="http://www.fehcom.de/ipnet/ucspi-ssl.html"
-#distfiles="http://www.fehcom.de/ipnet/ucspi-ssl/${pkgname}-${version}.tgz"
-distfiles="https://sources.voidlinux.eu/${pkgname}-${version}/${pkgname}-${version}.tgz"
-checksum=f056deb954f4396123cdd97c8c8adc6386a51b49b0be6e22a9eb4a24e43e7f69
-wrksrc="${pkgname}-${version}"
-create_wrksrc=yes
-build_wrksrc="host/superscript.com/net/${pkgname}-${version}/src"
+distfiles="http://www.fehcom.de/ipnet/ucspi-ssl/ucspi-ssl-${version}.tgz"
+checksum=b246a4cdab48a0a47854892ce683caf5444096af0c2b8ccd04e5b4d604e75633
 # ad hoc build system breaks parallel build and cross-compilation
 disable_parallel_build=yes
 nocross=yes
 
+# Use wget to bypass cert issue with upstream
+do_fetch() {
```

Comment:
use `export SSL_NO_VERIFY_PEER=yes`  instead of custom do_fetch and do_extract

Reply:
Thanks, I was trying to figure out how to do that